scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -300,6 +301,25 @@
          (catch Exception e
            (log/error e "Mesos scheduler status update error")))))
 
+(defn write-sandbox-url-to-datomic
+  "Takes a sandbox file server URL from the compute cluster and saves it to datomic."
+  [conn task-id sandbox-url]
+  (try (let [db (db conn)
+             _ (when-not task-id
+                 (throw (ex-info "task-id is nil. Something unexpected has happened."
+                                 {:sandbox-url sandbox-url
+                                  :task-id task-id})))
+             [instance] (first (q '[:find ?i
+                                    :in $ ?task-id
+                                    :where
+                                    [?i :instance/task-id ?task-id]]
+                                  db task-id))]
+         (if (nil? instance)
+           (log/error (str "Sandbox file server URL update error. No instance for task-id " task-id))
```

Comment:
```suggestion
           (log/error "Sandbox file server URL update error. No instance for task-id" task-id)
```
`str` isn't necessary since [`log/error`](http://clojure.github.io/tools.logging/#clojure.tools.logging/error) (and the other logging functions) behave like `println` (it joins all of the arguments with spaces).